2.6.md

```diff
@@ -827,7 +827,7 @@ This object describes the nature and behavior of the entity that is the source o
 
 ### 3.2.3 - Object: Regs <a name="objectregs"></a>
 
-This object contains any legal, governmental, or industry regulations that the sender deems applicable to the request. See Section 7.5 for more details on the flags supporting Coppa, GDPR and CCPA.
+This object contains any legal, governmental, or industry regulations that the sender deems applicable to the request. See Section 7.5 for more details on the flags supporting Coppa, GDPR and others.
```

Comment:
COPPA is also an acronym.

Reply:
I would delete the word 'any'; I'm not sure it adds any clarity.